### PR TITLE
Evaluation utility for geo models

### PR DESCRIPTION
**Pull request by sayalikulkarni**
Command line utility to get distance based evaluation metrics for geo models in RVS.

### CODE REVIEW

File: cabby/evals/eval_geo_model.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+import numpy as np
+from absl import app, flags
+from geopy.distance import great_circle
+
+from cabby import logger
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string("input_file", None,
+                    "TSV file containing true and predicted co-ordinates.")
```
Comment (tzufgoogle) [inline cabby/evals/eval_geo_model.py:38]: The model outputs the prediction and ground truth in a 'pt' format so a converstion to an 'tsv' is needed. Maybe add a function that does that?

File: cabby/evals/eval_geo_model.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+			Args:
+				error_distances: Array of distance errors.
+				eval_logger: Logger object.
+    """
+    num_examples = len(error_distances)
+    if num_examples == 0:
+      self.eval_logger.error("No examples to be evaluated.")
+    accuracy = len(np.where(error_distances == 0.)[0]) / num_examples
+    mean_distance, median_distance = np.mean(error_distances), np.median(
```
Comment (tzufgoogle) [inline cabby/evals/eval_geo_model.py:96]: maybe also calculate the max error just for a sanity check?

**Review by tzufgoogle — COMMENTED** [review]
A few suggestions.
(1) The format for the evaluation is different from what the model outputs. Maybe add a function to convert between the two?
(2) Maybe add a test that takes an output of the module and calculats the evaluation?

**Comment by sayalikulkarni** [comment]
abandoning this one.

**Pull request closed without merging**